### PR TITLE
[Styleguide] Move fixtures to /Fixtures directory 

### PR DESCRIPTION
**Pull request by damassi**
Was noticing when trying to bring pages into force that it doesn't like anything to come from `__storybooks__` folders. Moved fixtures out (which were referenced in a few places in our code) so that we can start presenting stuff in force. 

cc @oxaudo, @alloy, @zephraph, @mzikherman

### CODE REVIEW

File: .storybook/webpack.config.js [hunk 1]
```diff
@@ -84,6 +84,7 @@ module.exports = (baseConfig, env) => {
       extensions: [".mjs", ".js", ".jsx", ".ts", ".tsx"],
       alias: {
         sharify: sharifyPath.replace(/\.js$/, ""),
+        "styled-components": path.resolve("./node_modules/styled-components"),
```
Comment (alloy) [inline .storybook/webpack.config.js]: Why is this needed?

Comment (damassi) [reply .storybook/webpack.config.js]: It’s a multiple copies of lib error due to yarn link
